.golangci.yml

```diff
@@ -0,0 +1,4 @@
+run:
```

Comment:
`tests/e2e` is not the target of linter
https://github.com/cosmos/gaia/blob/v7.0.0/.golangci.yml#L1